edge-util/src/Microsoft.Azure.Devices.Edge.Storage/IBackupRestore.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage
+{
+    using System.Threading.Tasks;
+
+    public interface IBackupRestore
```

Comment:
>IBackupRestore [](start = 21, length = 14)

I think it should be a distinction in the name for IBackupRestore and IDbBackupRestore. IBackupRestore it seems the it contains the serialization algorithm, maybe add something like that to the name

Reply:
Renamed IBackupRestore to IDataBackupRestore and removed IDbBackupRestore and IDbStoreBackupRestore